


#include <stdlib.h>

#include <string.h>

#include <stdio.h>
#include <pb_encode.h>
#include <pb_decode.h>
#include "Txrequest.pb.h"
#include <pb.h>
#include "simple.pb.h"




#ifndef _BLOCKCHAINPB_H_
#define _BLOCKCHAINPB_H_








int blockchainpb(void * data,uint32_t datasize);
int simple();






#endif
